lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var writeAccessor = require('./writeAccessor');
+
+module.exports = cesiumGeometryToGltfPrimitive;
+
+// Get the first gltf attribute semantic of that type
+function getFirstAttributeSemantic(primitive, semantic) {
+    var attributes = primitive.attributes;
+    for (var attributeSemantic in attributes) {
+        if (attributes.hasOwnProperty(attributeSemantic)) {
+            if (attributeSemantic.replace(/[0-9]|_/g, '') === semantic) {
```

Comment:
This does work, but using `indexOf` is preferable since it doesn't create a new string.